### ISSUE TITLE
Inaccurate `main` value in `package.json`: causes "unable to resolve path" false positive

### ISSUE THREAD

**Issue by saxoncameron** [infiniteluke/react-reduce-motion#1]
Hi there, looks like the `main` key you have set in your `package.json` is set to `dist/react-reduce-motion.cjs.js`, whereas I notice in my `node_modules` folder only similarly named files exist:

<img width="396" alt="image" src="https://user-images.githubusercontent.com/3240185/68544461-5aa6ec00-0413-11ea-82b8-32b7b6d742df.png">

Therefore I assume this is a typo, and you're meaning to target `dist/react-reduce-motion.browser.cjs.js`.

This is a problem because linters may throw an error, such as the one I have now encountered: `Unable to resolve path to module 'react-reduce-motion'` where it's trying and failing to find the file referenced by `main`, and not finding it, thereby flagging the package as a broken path or uninstalled, even though it's a false positive.

**Comment by infiniteluke** [comment]
@saxoncameron Thanks for reporting this! What eslint rule is flagging this so that I can properly reproduce and fix?

**Comment by saxoncameron** [comment]
The rule in question is `import/no-unresolved` from `"eslint-plugin-import": "^2.17.3",`.

Looking more closely at your `package.json`, maybe I was mistaken in your intention since there's also the `browser: {}` object in there. Nonetheless, it's an unresolvable error - so something must be wrong!

Let me know if I can provide any more info.

**Comment by infiniteluke** [comment]
v2.0.2 fixes this @saxoncameron. Thanks for reporting and let me know if you see any other issues!

**Comment by saxoncameron** [comment]
Nice one - thanks Luke 👌

**Comment by infiniteluke** [comment]
@saxoncameron can I ask how you're using react-reduce-motion? Is there a public demo on the web?

**Comment by saxoncameron** [comment]
Hey mate! Unfortunately nothing public - it was used for a module in a private authenticated portal.

It's usage was for a complex use-case involving a multi-step form SPA and how I animated the transitions between "step 1" and "step 2" on the form, which was mostly JS logic, and I used this library to determine "reduced motion" for accessibility best practices. Typically for this sort of thing I use the CSS approach, but it was a niche use-case.

I foresee it also being useful for implementations of react-transition-group in some scenarios, but I'm just riffing now. Hope that helps!
